### PR TITLE
[CDAP-8311] Add group name to CLI create namespace command

### PR DESCRIPTION
**Pull request by anew**

### CODE REVIEW

File: cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java [hunk 1]
```diff
@@ -81,6 +81,7 @@
   NAMESPACE_NAME("namespace-name"),
   NAMESPACE_DESCRIPTION("description"),
   NAMESPACE_PRINCIPAL("principal"),
+  NAMESPACE_GROUPNAME("group-name"),
```
Comment (awholegunch) [inline cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java]: Should this not be
```
NAMESPACE_GROUP_NAME("group-name"),
```
to be consistent with the others?

**Comment by Nishith** [comment]
LGTM.

**Comment by awholegunch** [comment]
One comment, otherwise LGTM

**Comment by anew** [comment]
Addressed comment, squashed and rebased on develop.

**Comment by anew** [comment]
This only affects CLI and no other module depends on that. As CLI tests are passing, merging is ok at this time.

**Comment by anew** [comment]
btw, build is at http://builds.cask.co/browse/CDAP-DUT5390-1

**Pull request merged**